object/service.proto

```diff
@@ -83,7 +80,7 @@ message PutRequest {
         // Object with at least container id and owner id fields
         Object Object       = 1;
         // Token with session public key and user's signature
-        session.Token Token = 2;
+        service.Token Token = 2;
```

Comment:
Why `SearchRequest`, `GetRangeRequest`, `GetRangeHashRequest` don't have token field?

Reply:
They never had before. In the next changes, `Token` field will be removed from the custom fields of all requests due to the presence in `RequestVerificationHeader`.

Reply:
Got it.